packages/next/build/webpack/loaders/next-serverless-loader.ts

```diff
@@ -262,7 +262,7 @@ const nextServerlessLoader: loader.Loader = function() {
 
       const parsedUrl = handleRewrites(parse(req.url, true))
 
-      if (parsedUrl.pathname.match(/_next\\/data/)) {
+      if (parsedUrl.pathname && parsedUrl.pathname.match(/_next\\/data/)) {
```

Comment:
I'm not sure that this should ever be the case with a proper set-up. On ZEIT Now we ensure that the `req.url` is always valid. It looks like [this section of code](https://github.com/danielcondemarin/serverless-next.js/blob/8da92e2d8e23e361c24904e4579e9d4e02845126/packages/next-aws-lambda/lib/compatLayer.js#L15-L18) isn't making sure to set `req.url` to `/` when doing the replace so this makes me think the issue should be moved to that repo 🤔